src/android/app/src/main/java/org/citra/citra_emu/utils/Log.kt

```diff
@@ -4,34 +4,29 @@
 
 package org.citra.citra_emu.utils
 
-import android.util.Log
-import org.citra.citra_emu.BuildConfig
-
-/**
- * Contains methods that call through to [android.util.Log], but
- * with the same TAG automatically provided. Also no-ops VERBOSE and DEBUG log
- * levels in release builds.
- */
+import android.os.Build
+
 object Log {
     // Tracks whether we should share the old log or the current log
     var gameLaunched = false
-    private const val TAG = "Citra Frontend"
 
-    fun verbose(message: String?) {
-        if (BuildConfig.DEBUG) {
-            Log.v(TAG, message!!)
-        }
-    }
+    external fun debug(message: String)
 
-    fun debug(message: String?) {
-        if (BuildConfig.DEBUG) {
-            Log.d(TAG, message!!)
-        }
-    }
+    external fun warning(message: String)
+
+    external fun info(message: String)
 
-    fun info(message: String?) = Log.i(TAG, message!!)
+    external fun error(message: String)
 
-    fun warning(message: String?) = Log.w(TAG, message!!)
+    external fun critical(message: String)
 
-    fun error(message: String?) = Log.e(TAG, message!!)
+    fun logDeviceInfo() {
+        info("Device Manufacturer - ${Build.MANUFACTURER}")
```

Comment:
```suggestion
        info("[Log] Device Manufacturer: ${Build.MANUFACTURER}")
```
Nit.
To keep it consistent across the project.
Ditto for the logs below too

Reply:
I don't know if the CitraBot can parse this, if there's extra stuff at the beginning.

Reply:
I probably shouldn't change the format that Liu based the bot checks on. Since I have no way of checking his code, I will just leave this log as-is.